tests/conftest.py

```diff
@@ -60,6 +62,29 @@ def fixture_sample_tag_names(vcf_tag_name: str, sample_tag_name: str) -> List[st
     return [vcf_tag_name, sample_tag_name]
 
 
+@pytest.fixture(scope="function", name="spring_tag")
+def fixture_spring_tag() -> str:
+    """Return the tag marking spring files."""
```

Comment:
```suggestion
    """Return the tag marking SPRING files."""
```
below SPRING is capitalised.